save-backend/src/test/kotlin/org/cqfn/save/backend/controller/AgentsControllerTest.kt

```diff
@@ -16,14 +18,19 @@ import org.springframework.test.web.reactive.server.WebTestClient
 import org.springframework.test.web.reactive.server.expectBody
 import org.springframework.web.reactive.function.BodyInserters
 import java.time.LocalDateTime
+import javax.transaction.Transactional
 
 @SpringBootTest(classes = [SaveApplication::class])
 @AutoConfigureWebTestClient
 @ExtendWith(MySqlExtension::class)
+@Transactional
```

Comment:
Is it still needed? Also, I believe it's better to place Transactional on controller itself (though, it needs manual/integration testing)

Reply:
https://github.com/cqfn/save-cloud/pull/99/files#r617588302